Makefile.in

```diff
@@ -71,6 +71,7 @@ distdir:
 
 cstyle:
 	find $(top_srcdir) -name '*.[hc]' \
+                ! -wholename './src/config.h' \
```

Comment:
can you please change `wholename` to `path`, being more portable?

Reply:
Sure!